File: apollo/src/main/java/io/muun/apollo/domain/errors/InvalidSwapException.java
package io.muun.apollo.domain.errors;

import io.muun.common.exception.PotentialBug;

public class InvalidSwapException extends RuntimeException implements PotentialBug {

    public InvalidSwapException(String swapUuid) {
        super("Validation failed for swap UUID " + swapUuid);
    }
}

File: android/apolloui/src/main/java/io/muun/apollo/domain/errors/newop/InvalidSwapException.kt
package io.muun.apollo.domain.errors.newop

import io.muun.apollo.domain.errors.MuunError
import io.muun.common.exception.PotentialBug

class InvalidSwapException(swapUuid: String) : MuunError(
    "Validation failed for swap UUID $swapUuid"
), PotentialBug
